docs/sources/clients/k6/_index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: k6 load testing
+weight: 90
+---
+
+# k6 Loki extension load testing
+
+Grafana [k6](https://k6.io) is a modern load-testing tool.
+Its clean and approachable scripting [API](https://k6.io/docs/javascript-api/)
+works locally or in the cloud.
+Its configuration makes it flexible.
+
+The [xk6-loki extension](https://github.com/grafana/xk6-loki) permits pushing logs to and querying logs from a Loki instance.
+It acts as a Loki client,
+simulating real-world load test scenarios on your Loki installation.
+
+## Before you begin
+
+k6 is written in Golang. [Download and install](https://go.dev/doc/install) a Go environment.
+
+## Installation
+
+`xk6-loki` is an extension to the k6 binary.
+Build a custom k6 binary that includes the `xk6-loki` extension.
+
+1. Install the `xk6` extension bundler:
+
+   ```bash
+   go install go.k6.io/xk6/cmd/xk6@latest
+   ```
+
+1. Check out the `grafana/xk6-loki` repository:
+
+   ```bash
+   git clone github.com/grafana/xk6-loki
+   cd xk6-loki
+   ```
+
+1. Build k6 with the extension:
+
+   ```bash
+   make k6
+   ```
+
+## Usage
+
+Use the custom-built k6 binary in the same way as a non-custom k6 binary:
+
+```bash
+./k6 run test.js
+```
+
+`test.js` is a Javascript load test.
+Refer to the [k6 documentation](https://k6.io/docs/) to get started.
+
+### Scripting API
+
+The custom-built k6 binary provides a Javascript `loki` module.
+
+Your Javascript load test imports the module: 
+
+```js
+import loki from 'k6/x/loki';
+```
+
+Classes of this module are:
+
+| class | description |
+| ----- | ----------- |
+| `Config` | configuration for the `Client` class |
+| `Client` | client for writing and reading logs from Loki |
+
```

Comment:
Perhaps, it is good to mention here or in the example below that `loki.Config` and `loki.Client` must be called on the k6 init context (outside the `default` function). `k6` will configure the Loki client once and the client will be shared between all the VU iterations.

https://k6.io/docs/using-k6/test-life-cycle/


Same comment for https://github.com/grafana/xk6-loki